File: scripts/push-to-github.py
import subprocess
import sys

try:
    # Stage all changes
    print("[v0] Staging all changes...")
    subprocess.run(["git", "add", "."], cwd="/vercel/share/v0-project", check=True)
    
    # Check if there are changes to commit
    result = subprocess.run(
        ["git", "diff", "--cached", "--quiet"],
        cwd="/vercel/share/v0-project"
    )
    
    if result.returncode != 0:
        # Changes exist, commit them
        print("[v0] Committing changes...")
        subprocess.run(
            [
                "git",
                "commit",
                "-m",
                "Fix: Ensure tier/score consistency in entry decisions"
            ],
            cwd="/vercel/share/v0-project",
            check=True
        )
        
        # Push to GitHub
        print("[v0] Pushing to GitHub...")
        subprocess.run(
            ["git", "push", "origin", "v0/jaspalbilkhu-2038-ab520cb6-3"],
            cwd="/vercel/share/v0-project",
            check=True
        )
        print("[v0] ✓ Successfully pushed to GitHub!")
    else:
        print("[v0] No changes to commit")
        
except subprocess.CalledProcessError as e:
    print(f"[v0] Error: {e}", file=sys.stderr)
    sys.exit(1)
except Exception as e:
    print(f"[v0] Unexpected error: {e}", file=sys.stderr)
    sys.exit(1)

File: scripts/push-to-github.mjs
import { execSync } from 'child_process';

try {
  console.log('Adding all changes...');
  execSync('git add .', { stdio: 'inherit' });
  
  console.log('Creating commit...');
  execSync('git commit -m "fix: reconcile tier and score consistency in entry decisions"', { stdio: 'inherit' });
  
  console.log('Pushing to GitHub...');
  execSync('git push origin v0/jaspalbilkhu-2038-ab520cb6-3', { stdio: 'inherit' });
  
  console.log('✓ Changes pushed to GitHub successfully');
} catch (error) {
  console.error('Error pushing to GitHub:', error.message);
  process.exit(1);
}
